lib/insights/api/common/metrics.rb

```diff
@@ -33,6 +34,23 @@ def self.activate(config, prefix)
           require "insights/api/common/middleware/web_server_metrics"
           Rails.application.middleware.unshift(Insights::API::Common::Middleware::WebServerMetrics, :metrics_prefix => prefix)
         end
+
+        private_class_method def self.setup_custom_metrics(custom_metrics)
+          return if custom_metrics.nil?
+
+          custom_metrics.each do |metric|
+            instance_variable_set("@#{metric[:name]}_#{metric[:type]}", PrometheusExporter::Client.default.register(metric[:type], metric[:name], metric[:description]))
```

Comment:
Is there any way to validate or map a list of values that `:name, :type, :description` can be? Potentially like `constantize_with_care`

In this current form this seems like we'll `eval` any of the code keys coming in and that seems like potential for a problem.